app/src/main/java/com/android/shelfLife/model/newRecipe/RecipeRepositoryFirestore.kt

```diff
@@ -247,13 +253,25 @@ class RecipeRepositoryFirestore @Inject constructor(private val db: FirebaseFire
           instructions = instructions,
           servings = servings,
           time = time,
-          ingredients = ingredients.map { convertToIngredient(it) })
+          ingredients = ingredients.map { convertToIngredient(it) },
+          recipeType = convertToSearchType(recipeType))
     } catch (e: Exception) {
       Log.e("RecipeRepository", "Error converting document to Recipe", e)
       null
     }
   }
 
+  private fun convertToSearchType(recipeType: String): RecipeType {
```

Comment:
Its good to have enum types to represent the types of recipes, I approve but you should also have a deafult type just incase